### PR TITLE
Integrate styled components admin

### LINKED ISSUE #2196: Integrate Styled Components in Admin
**Actual Behaviour**
Not present.

**Expected Behaviour**
Have a common styled component to avoid repetition of code.

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by harsh-jindal**
Fixes #2196 
Changes: Left ListUser as data is not loading.

Demo Link: https://pr-2209-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

File: src/components/Admin/ListSkills/ListSkills.js [hunk 1]
```diff
@@ -26,10 +26,26 @@ import {
...
+`;
+
+const ActionDiv = styled.div`
+  cursor: pointer;
+  color: #49a9ee;
+`;
+
+const Container = styled.div`
+  display: flex; 
+  flexD-direction: column';
```
Comment (fragm3) [inline src/components/Admin/ListSkills/ListSkills.js]: New CSS property I suppose 🤔

Comment (harsh-jindal) [reply src/components/Admin/ListSkills/ListSkills.js]: 😆 Will correct it.

**Comment by harsh-jindal** [comment]
@fragm3 Made the changes

**Comment by fragm3** [comment]
@harsh-jindal this is not what I meant, use`css` provided by styled-components.

**Comment by harsh-jindal** [comment]
@fragm3 Shall I create a new file called AdminStyle.js ?

**Comment by fragm3** [comment]
@harsh-jindal` import {css} from styled-components` and use it for declaring styles like:

`const commonStyles = css`display: none`;`

Use it in styled components like:

```
const Button = styled(_Button)`
   ${commonStyles}
  Other specific styles
`
```

If there are common styles, feel free to create `AdminStyles.js`

**Comment by harsh-jindal** [comment]
@akshatnitd @fragm3 Made the changes.

**Pull request merged**